doc/birds.md

```diff
@@ -9,9 +9,9 @@ Some people consider it reasonable to name [combinators](primitive.md#modifiers)
 | `⊣`     | Identity  | `I`   | Kestrel      | `K`
 | `⊢`     | Identity  | `I`   |              | `KI`
 | `∘`     | Bluebird  | `B`   | Blackbird    | `B₁`
-| `○`     | Bluebird  | `B`   | Psi?         | `D₂`-like: `labcd.a(bc)(bd)`
+| `○`     | Bluebird  | `B`   | Psi          | `ψ`
 | `˙`     | Kestrel   | `K`   |              | `KK`
-| `⊸`     |           | `R*S` | Queer        | `Q`
+| `⊸`     |           | `BSC` | ~Dove        | `D`-like: `labcd.a(bc)d`
 | `⟜`     | Starling  | `S`   | ~Dove        | `D`-like: `labcd.ac(bd)`
```

Comment:
Sorry about the confusion. Some times reviews are easier by create comments / reviews on the code (in this case markdown). 

> Monadic ⊸ is definitely l𝔽𝔾𝕩.𝔾(𝔽𝕩)𝕩, or labc.b(ac)c. It applies 𝔽 before applying 𝔾, that's where the name comes from.

Is this consistent with monadic ⟜ being `labc.ac(bc)`? Seems like we are switching the arity of `a` and `b` in then.

Reply:
Yea, I think I see what you mean.
```
(≡⟜⌽)¨"cat"‿"tacocat" # a is ≡, b is ⌽
(⌽⊸≡)¨"cat"‿"tacocat" # b is ≡, a is ⌽ ?
```
I can see why you like `b(ac)c` but I think you could argue `a` should just be the binary function in both cases. You could have just as easily spelt it `≡⊸⌽` (although I agree that putting it on the side of the binary function which corresponds to the argument that is being "preprocessed" makes more sense).

Reply:
Got it :) 
```hs
Prelude> let s a b c = a c (b c)
Prelude> let c = flip
Prelude> let b = (.)
Prelude> let b1 a b c d = a (b c d)
Prelude> :t b1 c b s c
b1 c b s c :: (t1 -> t2) -> (t2 -> t1 -> t3) -> t1 -> t3
```
This means Monadic ⊸ is `B1CBSC` and corresponds to `labc.b(ac)c`